Cargo.toml

```diff
@@ -26,20 +26,19 @@ jsonwebtoken = "~7.2"
 lazy_static = "~1.4"
 log = "~0.4"
 percent-encoding = "~2.1"
-protobuf = { version = "^2.0", features = [ "with-bytes" ] }
+protobuf = { version = "^2.20", features = [ "with-bytes" ] }
 # default features force a dependency on openssl-sys, which we want to avoid for embedded applications
 publicsuffix = { version = "~1.5.4", default-features = false }
 quick-error = "~2.0"
 rand = "~0.7"
 rand_chacha = "~0.2.2"
 recrypt = "~0.11"
 regex = "~1.4"
-reqwest = { version = "~0.10.0", features = [ "json" ], default-features = false }
+reqwest = { version = "~0.11.0", features = [ "json" ], default-features = false }
 ring = { version = "~0.16", features = [ "std" ] }
 serde = { version = "~1.0", features = [ "derive" ] }
 serde_json = "~1.0"
-# ironoxide requires rt-threaded at runtime, but not at compile time
-tokio = { version = "~0.2.0", features = [ "time", "rt-threaded" ] }
+tokio = { version = "~1.0", features = [ "time", "rt-multi-thread" ] }
```

Comment:
Maybe we only need `rt-multi-threaded` for the `blocking` feature?

Reply:
I think this is true? I was a little wary because of the comment that you had written about requiring `rt-threaded` at runtime. But if I depend on local ironoxide, ironoxide-cli still works fine (after a quick upgrade of tokio), so I think this means that it's all good to make it an optional flag?

Reply:
It looks like I can also remove the `time` flag altogether? Or is there some other test I'd have to do to check that

Reply:
If the tests and benches work without it, we don't need it.

Reply:
`rt-threaded` previously made all of tokio use the multi-threaded scheduler vs the single threaded one. Seems like `rt-multi-thread` is the new name of it, pretty sure we want it, but I don't know.

Reply:
We do need it for timeouts, so we added it back in. We were perhaps picking it up (transitively) via reqwest.

Reply:
As for `rt-multi-threaded`, there's nothing in the library that requires it, but it seems like most consumers would want it. I suggest we make it a default feature so it could be disabled if needed.